objective.py

```diff
@@ -18,7 +18,7 @@ class Objective(BaseObjective):
 
     install_cmd = "conda"
     requirements = [
-        "torch",
+        "pytorch",
```

Comment:
If we have issues with `torch` or `pytorch`, we can switch to `pip:torch` for the tests. If someone needs more granular control, they should `conda install pytorch` anyway.

Reply:
Let me check if that changes anything